README.md

```diff
@@ -1,193 +1,481 @@
+  >
+  >The decision of this change is made on the assumption that Solr may not be required in the context of files tab search, whose search facets are reduced compared to other in-application searches. Therefore, if we find evidence that the assumption is incorrect, we will work on extending the search capabilities to support Solr.
+
+</details>
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
 
-- 2023-08-01: [View mode of the dataset page](https://groups.google.com/g/dataverse-community/c/cxZ3Bal_-uo/m/h3kh3iVNCwAJ)
-- 2023-12-13: [Files table on the dataset page](https://groups.google.com/g/dataverse-community/c/w_rEMddESYc/m/6F7QC1p-AgAJ)
 
 ## Getting Started
 
-First install node >=16 and npm >=8. Recommended versions `node v19` and `npm v9`.
+_To get a local copy up and running follow these simple example steps._
 
-### Create a `.npmrc` file and add a token
+### Prerequisites
 
-To install the [@iqss/dataverse-client-javascript](https://github.com/IQSS/dataverse-client-javascript/pkgs/npm/dataverse-client-javascript)
-from the GitHub registry, necessary for connecting with the Dataverse API, follow these steps to create an `.npmrc` file in
-the root of your project using your GitHub token.
+[![DockerDesktop][_shield_docker]][_uses_docker_url]
 
-1. **Copy `.npmrc.example`**
+1. **Node &amp; NPM**: `node >= v16` and `npm >= v8`. Recommended versions for this project are `node v19` and `npm v9`.
+2. **Docker**: We use Docker Desktop, but the Docker CLI will also work.
+3. **Create a Copy of .npmrc**: In the project directory root, duplicate `.npmrc.example`, saving the copy as `.npmrc`.
 
-   Duplicate the `.npmrc.example` file in your project and save it as `.npmrc`.
+    >   ```bash
+    >   # root project directory
+    >   cp .npmrc.example .npmrc
+    >   ```
 
-2. **Replace the Token**
+4. **Create and Add Your Npm and GitHub Tokens**: In order to connect with the Dataverse API, developers will need to install [@iqss/dataverse-client-javascript][dv_repo_dvclientjs_npm_url] from the GitHub registry by following the steps outlined below. Read more about access tokens on [GitHub Docs][dv_docs_github_userauthtoken_url].
 
-   Open the newly created `.npmrc` file and replace `YOUR_GITHUB_TOKEN` with your actual GitHub token.
+    >  **Getting a GitHub Token**
+    >  1. Go to your GitHub [Personal Access Tokens][dv_docs_github_token_url] settings
+    >  2. Select `Generate new token (classic)`
+    >  3. Give the token a name and select scope `read:packages`
+    >  4. Copy the generated token and replace the string `YOUR_GITHUB_AUTH_TOKEN` in the previously created `.npmrc` file.
+    >       Now, you should be able to install the [Dataverse JavaScript][dv_repo_dvclientjs_url] client using npm.
 
-   ```plaintext
-   legacy-peer-deps=true
+Afterwards, your .npmrc file should resemble the following:
```

Comment:
@GermanSaracca comment
> Should we add a note? saying that the .npmrc file is still not identical as .npmrc.example, but because we will cover that later in the readme ( I think you already cover that in the design system deployment readme right?)
> Or it will be too mucht text?

https://github.com/IQSS/dataverse-frontend/pull/349#discussion_r1530207386

Reply:
Change addressed in the Developer Guide, check this comment
https://github.com/IQSS/dataverse-frontend/pull/353#discussion_r1546392871